antiplag/views.py

```diff
@@ -124,3 +123,55 @@ def get(self, request, id):
         else:
             # unprocessed submission documents should 'not exist' for the user
             return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+class DocumentDiff(APIView):
+    def get(self, request, first_id, second_id):
+        first_document = get_object_or_404(Document, pk=first_id)
+        second_document = None
+        intervals = []
+
+        if first_document.submission.status == Submission.SubmissionStatus.PROCESSED:
+
+            for doc in first_document.result.matched_docs:
+                if doc["elastic_id"] == second_id:
+                    second_document = doc
+
+            intervals = second_document["intervals"]
+
+            if not second_document:
```

Comment:
I think this should be above `intervals`, shouldn't it?

Reply:
fixed